test_module/__test__/tokio_readfile.spec.js

```diff
@@ -0,0 +1,20 @@
+const test = require('ava')
+const path = require('path')
+const bindings = require('../index.node')
+
+const filepath = path.resolve(__dirname, './example.txt')
+
+test('should read a file and return its a buffer', async (t) => {
+  return new Promise((resolve, reject) => {
+    bindings.testTokioReadfile(filepath, (err, value) => {
+      try {
+        t.is(err, null)
+        t.is(Buffer.isBuffer(value), true)
+        t.is(value.toString(), 'Hello world!\n')
```

Comment:
Maybe `t.is(value.toString(), fs.readFileSync(filepath, 'utf8'))` could fix this case in windows.